website/content/accessibility/deaf-and-hard-of-hearing/conference-best-practices.md

```diff
@@ -180,13 +186,13 @@ If speakers need interpreters or caption monitors, consider facilitating dry run
 
 Introduce deaf/hoh speakers to sign language interpreters prior to the event. This will allow them to share materials (slides, speaker notes, etc.) with interpreters. Knowing what the talk is about will help provide much more accurate interpretation services. 
 
-Signs for different technologies are not yet fully established (that is especially true in cloud native). Speakers and interpreters will need to agree on how to sign key terms and concepts.  Agreeing on a shared vocabulary for on-stage communications will enhance communication efficiency.
+Signs for different technologies are not yet fully established (that is especially true in cloud native). Speakers and interpreters will need to agree on how to sign key terms and concepts. Agreeing on a shared vocabulary for on-stage communications will enhance communication efficiency.
 
 If there are non-signing deaf or hard of hearing speakers, place a monitor visible from the stage so they can read the captions from fellow panelists and during Q&A. Note: captions on the screen facing the audience won't work in this case, they will need a monitor facing the speakers.
  
 ### Microphones for speakers and Q&A
 
-Provide microphones for speakers and audience questions — even in small rooms. Hard of hearing attendees or speakers will struggle to hear the question without a microphone, and the captions tool won't be able to capture it either. If you can't provide a microphone for audience questions, have the speaker or someone else repeat the question in the microphone, "This person was asking …" 
+Provide microphones for speakers and audience questions — even in small rooms. Hard of hearing attendees or speakers will struggle to hear the question without a microphone, and the captions tool won't be able to capture it either. If you can't provide a microphone for audience questions, have the speaker or someone else repeat the question in the microphone: "This person was asking …" 
```

Comment:
could you add this as a suggestion? Not really sure I understand

Reply:
Sure. Let me try to explain. The line `195` in itself is composed of multiple lines making it way too long to be a single line. Here's a screenshot of line `195` in which just a `,` has been replaced with `:`, but then when you see the diff, it sort of shows you `5` lines. That's because we are using very long line. So the ask here is to split all long lines into shorter ones. That's what I meant by `deliberate` line breaks. Like limiting a line to say 80 or 100 chars per line and breaking in into new line for subsequent words/characters. 

Makes sense now? We'd need to do this in the entire doc. 

<img width="824" alt="git_diff" src="https://github.com/cncf/tag-contributor-strategy/assets/5564497/115da37f-1c71-4e0f-8128-0462eb8686e0">

Reply:
Ok, that means we'd need to update the guidelines for publishing content on the website. Honestly, I prefer keeping sentences together. The changes are easy to see because they are highlighted. We use an 80-character limit for the linkerd.io blog, and it splits sentences up. When editing, you often edit an entire sentence, so having several lines for one sentence is more inconvenient to edit, in my view. In any case, it's a bigger discussion. If you have strong feelings about it, let's discuss it on Slack. I'll mark this as resolved in the meantime.

Reply:
Sure Catherine. Agree, this is part of bigger discussion and not a high prio one. So good to mark it as resolved.